### ISSUE TITLE
Accessibility - duplicate IDs on authorization page

### ISSUE THREAD

**Issue by AndreiAfrasinei** [doorkeeper-gem/doorkeeper#1541]
### Context
We're working to make our application accessible such that it meets the [WCAG - Web Content Accessibility Guidelines](https://www.w3.org/TR/WCAG20-TECHS/). 

### Steps to reproduce
If we access the web page rendered by the [authorization/new.html.erb file](https://github.com/doorkeeper-gem/doorkeeper/blob/main/app/views/doorkeeper/authorizations/new.html.erb) and we inspect the submit buttons we find duplicate IDs that are causing [this guideline failure](https://www.w3.org/TR/WCAG20-TECHS/F77.html). To replicate this you can paste the HTML from the previously mentioned file into [this HTML checker tool](https://validator.w3.org/nu/#textarea).

The reason why this happens is that the page has the same ID on the fields generated by the **hidden_field_tag** for the 2 buttons (authorize and deny)
(I am attaching a screenshot here of the 2 HTML forms that have fields with duplicated IDs)
![Selecție_098](https://user-images.githubusercontent.com/22632229/143270083-e90f9cd1-9fe0-42b2-af7c-fa263a93dd53.png)

### Expected behavior
The HTML page should not have duplicated IDs.

### Actual behavior
The HTML page raises an accessibility error due to duplicated IDs.
![Selecție_097](https://user-images.githubusercontent.com/22632229/143227984-10e0ab9d-7fa7-4f95-bcb3-2038629b3ca1.png)

### Suggested solution
A solution would be to change the **id** attribute form the HTML generated to **class**
In order to do that I would suggest instead of using **hidden_field_tag** - you could use something like: 
```ruby
<input type="hidden" name="client_id" class="client_id" value="<%= @pre_auth.client.uid %>"> 
```

This will change the HTML into this:
![Selecție_099](https://user-images.githubusercontent.com/22632229/143270362-c962e12b-9172-4663-b7e8-1d1d139876ce.png)

Which would make this page meet the accessibility guidelines.

**Comment by nbulaj** [comment]
Hi @AndreiAfrasinei . Thanks for reporting this!
I don't remember if we are coupled somehow to HTML IDs/classes (I personally don't remember any JS event handlers or CSS rules for them - https://github.com/doorkeeper-gem/doorkeeper/tree/main/app/assets/stylesheets/doorkeeper ) so I think it's safe to do such changes.

Would you like to propose a PR?

**Comment by AndreiAfrasinei** [comment]
Thank you for your response @nbulaj 

I will do the changes and open a PR in the near future - thanks again!
